src/browser/preload_scripts.ts

```diff
@@ -102,11 +93,7 @@ export function fetchAndLoadPreloadScripts(
             return Promise.resolve(true);
         } else {
             // not previously downloaded *OR* previous downloaded failed
-            if (isPreloadScript(preload)) {
-                return fetchToCache(identity, preload).then(loadFromCache);
-            } else {
-                return fetchPlugin(identity, preload).then(loadFromCache);
-            }
+            return fetchToCache(identity, preload).then(loadFromCache);
```

Comment:
Is the fail case something we may need to handle?

Reply:
This one was a quick edit for other changes in this PR without changing internal functions. We can do it, but in a separate PR though